src/install/cassandra/intro.mdx

```diff
@@ -0,0 +1,34 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+Our Cassandra integration sends performance [metrics](#metrics) and [inventory data](#inventory) from your Cassandra database to the New Relic platform. You can view pre-built dashboards of your Cassandra metric data, create alert policies, and create your own custom queries and charts.
+
+Read on to install the integration and to see what data we collect.
+
+## Quick start [#quick]
+
+Instrument your Cassandra database quickly and send your telemetry data with guided install. Our guided install uses our infrastructure agent and our CLI to set up the Cassandra integration, and discovers other applications and log sources running in your environment and then recommends which ones you should instrument. The guided install works with most setups. But if it doesn't suit your needs, there are other install options below.
```

Comment:
I'd put this section in a callout. Currently the intro looks a bit complicated and hard to parse, but putting this "optional" section in a callout helps break it up.

Reply:
Done!